.circleci/config.yml

```diff
@@ -1,48 +1,47 @@
-version: 2
+version: 2.1
 
-defaults: &defaults
-  working_directory: ~/repo
-  docker:
-    - image: circleci/node:lts
-cache_key: &cache_key
-  key: dependency-cache-lts-{{ checksum "package.json" }}
+aliases:
+  - &cache-key dependency-cache-{{ checksum "package.json" }}
+  - &cache-key-fallback dependency-cache-
+ 
+executors:
+  default:
+    working_directory: ~/repo
+    docker:
+      - image: cimg/node:lts
 
 jobs:
   checkout_code:
-    <<: *defaults
+    executor: default
```

Comment:
I am guessing `default` is not a special keyword in here (e.g. the default executor when nothing was specified) and it is treated as just a name

Reply:
Yes, it's not a special keyword, just a reference to the executor.